lightblue-migration-monitor/src/main/java/com/redhat/lightblue/migrator/monitor/Monitor.java

```diff
@@ -0,0 +1,93 @@
+package com.redhat.lightblue.migrator.monitor;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.joda.time.Period;
+import org.joda.time.format.PeriodFormat;
+import org.joda.time.format.PeriodFormatter;
+
+import com.redhat.lightblue.client.LightblueClient;
+import com.redhat.lightblue.client.LightblueException;
+import com.redhat.lightblue.client.Literal;
+import com.redhat.lightblue.client.Projection;
+import com.redhat.lightblue.client.Query;
+import com.redhat.lightblue.client.http.LightblueHttpClient;
+import com.redhat.lightblue.client.request.data.DataFindRequest;
+import com.redhat.lightblue.client.response.LightblueDataResponse;
+import com.redhat.lightblue.migrator.MigrationConfiguration;
+import com.redhat.lightblue.migrator.MigrationJob;
+
+public class Monitor {
+
+    private final LightblueClient lightblueClient;
+
+    public Monitor(MonitorConfiguration cfg) {
+        if (cfg.getClientConfig() != null) {
+            lightblueClient = new LightblueHttpClient(cfg.getClientConfig());
+        } else {
+            lightblueClient = new LightblueHttpClient();
+        }
+    }
+
+    public void runCheck(final Notifier... notifiers) throws LightblueException {
+        List<String> configurationsMissingJobs = new ArrayList<>();
+        for (MigrationConfiguration cfg : findMigrationConfigurations()) {
+            long period = parsePeriod(cfg.getPeriod());
+            Date endDate = new Date();
+            Date startDate = new Date(endDate.getTime() - period);
```

Comment:
I assume you mean 2 or 3 periods ago? If so, fine with me, but what if the period is a day? Then the check wouldn't be for 2 or 3 days ago... are we ok with that?